sdk/trace/src/main/java/io/opentelemetry/sdk/trace/SdkTracer.java

```diff
@@ -28,9 +28,13 @@ public SpanBuilder spanBuilder(String spanName) {
       spanName = FALLBACK_SPAN_NAME;
     }
     if (sharedState.hasBeenShutdown()) {
-      return TracerProvider.noop()
-          .get(instrumentationLibraryInfo.getName(), instrumentationLibraryInfo.getVersion())
-          .spanBuilder(spanName);
+      Tracer tracer =
+          instrumentationLibraryInfo.getVersion() != null
+              ? TracerProvider.noop()
+                  .get(
+                      instrumentationLibraryInfo.getName(), instrumentationLibraryInfo.getVersion())
+              : TracerProvider.noop().get(instrumentationLibraryInfo.getName());
```

Comment:
good catch.

Reply:
But doesn't noop ignore all that anyway? Couldn't we just always pass the name?

Reply:
```suggestion
      Tracer tracer = TracerProvider.noop().get(instrumentationLibraryInfo.getName());
```

Reply:
yes, this is correct. I remember looking at this and thinking the same, but somehow I forgot to write it down. thanks!